script/test

```diff
@@ -35,4 +33,4 @@ export BOTO_CONFIG=/dev/null
 
 PYTHON=${PYTHON:-python}
 # run tests
-$PYTHON -m pytest --no-success-flaky-report -q $PYTEST_OPTIONS "$@"
+pytest --no-success-flaky-report -q $PYTEST_OPTIONS "$@"
```

Comment:
(for next PR: https://pypi.org/project/pytest-env/ and the PYTEST_OPTIONS should rather be in pyproject.toml, as `addopts`)